app/services/search/feed_content.rb

```diff
@@ -23,11 +23,30 @@ def search_documents(params:)
       def prepare_doc(hit)
         source = hit.dig("_source")
         source["tag_list"] = hit["tags"]&.map { |t| t["name"] } || []
+        source["id"] = hit.dig("_source", "id").split("_").last.to_i
+        source["tag_list"] = hit.dig("_source", "tags")&.map { |t| t["name"] } || []
+        source["flare_tag"] = hit.dig("_source", "flare_tag_hash")
```

Comment:
small optimization:

```ruby
source["id"] = source["id"].split // and so on
```

on line 24 we already `dig` out the `_source` attribute